anastruct/fem/postprocess.py

```diff
@@ -93,24 +93,31 @@ def __init__(self, system: "SystemElements"):
         self.system = system
 
     @staticmethod
```

Comment:
@staticmethod should be removed if `self ` is used.

Reply:
Fixed